### ISSUE TITLE
Unable to Access AJAX Error

### ISSUE THREAD

**Issue by mrUlrik** [olifolkerd/tabulator#2046]
**Describe the bug**
I am attempting to work with a JSON payload that occurs if API errors show up. It's coming from a Microsoft standard response from .NET Framework that includes properties Message, ExceptionMessage, ExceptionType, and StackTrace. I'd like to use ExceptionMessage.

However, using `ajaxError: function(xhr, textStatus, errorThrown)` textStatus and errorThrown are returning undefined. I have been unable to piece together using the XHR object, but that may be my experience using XHR.

**Tabulator Info**
- Which version of Tabulator are you using? 4.2.7
- Post a copy of your construct object if possible so we can see how your table is setup
```
 var table = new Tabulator("#table", {
 	ajaxError: function(xhr, textStatus, errorThrown) {
  	alert(textStatus);
    alert(errorThrown);
  },
  layout: "fitColumns",
  columns: [
  	{ title: "Not a Thing", field: "text" }
  ]
 });
 
 table.setData("https://httpstat.us/500");
```

**Working Example**
Please include a JS Fiddle or Code Pen that demonstrates the problem, it is very hard to diagnose an issue from a simple description
http://jsfiddle.net/m7av4c3z/1/

**To Reproduce**
1. Attempt to load data from an API that returns a server error
2. Access the error
3. Find it is undefined

**Expected behavior**
I'd like to read the contents of/work with the JSON response returned

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 74

**Smartphone (please complete the following information):**

**Additional context**

**Comment by aklaver** [comment]
Looks like the docs maybe out of sync with reality. When I look in ajax.js I see:
https://github.com/olifolkerd/tabulator/blob/4.2/src/js/modules/ajax.js

.catch((error)=>{
				console.error("Ajax Load Error: ", error);
				self.table.options.ajaxError.call(self.table, error);

				self.showError();

				setTimeout(function(){
					self.hideLoader();
				}, 3000);

				self.loading = false;

reject();

No reference to 'textStatus, errorThrown'. Looking in the browser console when running your Fiddle confirms this. I see:
Ajax Load Error: TypeError: NetworkError when attempting to fetch resource.
Looks like you have just the xhr to work with.

**Comment by olifolkerd** [comment]
Hey @aklaver 

Thanks for the info, i will make sure the documentation gets updated.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @aklaver 

i have pushed a fix for this and will bring it live when 4.3 is released later this evening

Cheers

Oli :)
